examples/rendercontrol_opengl/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Gui OpenGL Qml Quick)
+
+set(SOURCES
+    rendercontrol.qrc
```

Comment:
You should add one line `../images.qrc` here. Otherwise the svg images used by the system buttons can't be found.

Reply:
Fixed in 716815fe7743b561523a452771cdae39fb684f35.